sessionctx/variable/tidb_vars.go

```diff
@@ -51,6 +51,12 @@ const (
 	// tidb_opt_distinct_agg_push_down is used to decide whether agg with distinct should be pushed to tikv/tiflash.
 	TiDBOptDistinctAggPushDown = "tidb_opt_distinct_agg_push_down"
 
+	// tidb_broadcast_join_threshold_size is used to limit the size of small table for mpp broadcast join.
```

Comment:
The comment is really hard to understand.
I do not know the difference between `size` and `count`